charts/generic/pod-memory-hog/pod-memory-hog.chartserviceversion.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  createdAt: 2019-01-07T10:28:08Z
+  name: pod-memory-hog
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: CNCF
+    support: https://slack.kubernetes.io/
+spec:
+  displayName: pod-memory-hog
+  categoryDescription: |
+    Pod-Memory-Hog contains chaos to consume Memory resouces of specified containers in Kubernetes pods.  
+    - Causes high Memory resource consumption using dd command which maintains a temporary buffer in memory to write the chunk of data assigned in bs of the dd command.
+    - The application pod should be healthy once chaos is stopped. Expectation is that service-requests should be served despite chaos.
+  keywords:
+    - Kubernetes
+    - Memory
+  platforms:
+    - GKE
+    - Packet(Kubeadm)
+    - Minikube
+    - EKS
```

Comment:
Can you please verify for EKS?

Reply:
Its running fine in EKS also @ispeakc0de

Reply:
great